.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/aave-helpers"]
 	path = lib/aave-helpers
 	url = https://github.com/bgd-labs/aave-helpers
+[submodule "lib/stake-token"]
```

Comment:
plz remove the submodule.

the issue with submodules is that foundry is not quite good in detecting what is needed and what isn't so `forge build --force` will include everything on lib/stake-token and every dependency in that tree.
As you only need the interface plz just use it from here: https://github.com/bgd-labs/aave-address-book/blob/main/src/common/IStakeToken.sol